source/production/F0.Minesweeper.Server/Pages/Index.razor.cs

```diff
@@ -1,10 +1,6 @@
-using F0.Minesweeper.Components.Abstractions;
-using F0.Minesweeper.Logic.Abstractions;
-
 namespace F0.Minesweeper.Server.Pages
 {
 	public partial class Index
```

Comment:
Since this is now empty: do we want to remove code behind files that have right now nothing valuable?

Reply:
yes! :)
(That means the index hosts only the game component now?!)

Reply:
Not empty anymore due to the automatic redirect :D